dev/extra-ml-requirements.txt

```diff
@@ -11,7 +11,7 @@ h5py<3.0.0
 # Required by mlflow.sklearn
 scikit-learn
 # Required by mlflow.gluon
-mxnet
+mxnet!=1.8.0
```

Comment:
Fix for this error:

https://github.com/mlflow/mlflow/runs/2163733962#step:5:3141